packages/preview/cnam-template/1.0.0/typst.toml

```diff
@@ -0,0 +1,28 @@
+# for a description of available keys, see https://github.com/typst/packages/?tab=readme-ov-file#package-format
+
+[package]
+name = "cnam-template"
+version = "1.0.0"
+entrypoint = "src/lib.typ"
+authors = [
+  "Tom Planche <tomplanche@proton.me>",
+]
+license = "MIT"
+description = "A modular and organized TYPST template for creating professional documents using CNAM branding and styling."
```

Comment:
The word "Typst" is somewhat redundant here and should be omitted. See https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules